src/components/Select.tsx

```diff
@@ -41,7 +41,12 @@ function Select({
             aria-expanded="false"
             data-offset="10,20"
           >
-            {currentOption?.displayName || 'All'}
+            <span
+              className="d-inline-block"
+              style={{maxWidth: '85%', overflowX: 'clip', textOverflow: 'ellipsis'}}
```

Comment:
Bootstrap didn't have utility classes for a lot of these so I had to inline them